library/general/src/lib/installation/proposal_client.rb

```diff
@@ -132,6 +137,11 @@ def run
     #     The caller will format each item as a HTML list item (`<li>`). The
     #     proposal can contain hyperlinks with IDs listed in the list `links`.
     #
+    #   * **`"label_proposal"`** [Array<String>, nil]
```

Comment:
I would remove details from documentation and use it like
```
raw_proposal: Human readable proposal, not in final format. Each item is used as separate statement about module. For cramped_label_mode does not use any richtext tags. The proposal outside of cramped_label_mode can contain hyperlinks with IDs listed in the list links.
```

And to be honest I am not sure if final sentence do not change in future as it is quite useful to have one click modification.

about implementation I worry we need to have separate implementation even now due to links stuff and others.

Or just idea that comes to my mind is to not have any cramped_label_mode and simply always return label_proposal in response. Which can simplify a bit stuff, but also produce some duplicite work. What do you think?